src/features/mentors/edit-mentor/edit-mentor.js

```diff
@@ -64,7 +83,11 @@ export const EditMentor = ({ id }) => {
     error: deletedIsError,
   } = useSelector(mentorDeletingSelector, shallowEqual);
 
-  const [updateMentor, removeMentor, dispatchAddAlert] = useActions([editMentor, deleteMentor, addAlert]);
+  const [updateMentor, removeMentor, dispatchAddAlert] = useActions([
+    editMentor,
+    deleteMentor,
+    addAlert,
+  ]);
   const [groups, setGroups] = useState(mentorGroups || 0);
```

Comment:
it's not understandable what the initial state and it's type: number or array??

Reply:
I have not changed anything in this place

Reply:
I see, its not your changes